DesignSystem/Sources/TextField/MOITTextField.swift

```diff
@@ -0,0 +1,113 @@
+//
+//  MOITTextField.swift
+//  DesignSystem
+//
+//  Created by 최혜린 on 2023/06/05.
+//  Copyright © 2023 chansoo.MOIT. All rights reserved.
+//
+
+import UIKit
+
+import ResourceKit
+
+import FlexLayout
+import PinLayout
+import RxCocoa
+import RxSwift
+
+public final class MOITTextField: UIView {
+
+	private let flexRootView = UIView()
+	private var titleLabel: UILabel = {
+		let label = UILabel()
+		label.textColor = ResourceKitAsset.Color.gray700.color
+		label.font = ResourceKitFontFamily.p2
+		return label
+	}()
+	fileprivate var textField: UITextField = {
+		let textField = UITextField()
+		textField.layer.cornerRadius = 12
+		textField.clipsToBounds = true
+		textField.layer.borderColor = ResourceKitAsset.Color.gray200.color.cgColor
+		textField.layer.borderWidth = 1
+		textField.backgroundColor = ResourceKitAsset.Color.gray50.color
+		textField.font = ResourceKitFontFamily.p1
+		textField.textColor = ResourceKitAsset.Color.black.color
+		textField.tintColor = ResourceKitAsset.Color.black.color
+		textField.addHorizontalPadding()
+		return textField
+	}()
+
+	private let disposeBag = DisposeBag()
+	private let title: String
+	private let placeHolder: String
+
+// MARK: - init
+	public init(
+		title: String,
+		placeHolder: String
+	) {
+		self.title = title
+		self.placeHolder = placeHolder
+
+		super.init(frame: .zero)
+
+		configureLayout()
+		configureComponent()
+	}
+
+	required init?(coder: NSCoder) {
+		fatalError("init(coder:) has not been implemented")
+	}
+
+// MARK: - override
+	override public func layoutSubviews() {
+		super.layoutSubviews()
+
+		flexRootView.pin.all()
+		flexRootView.pin.height(85)
+		flexRootView.flex.layout()
+	}
+
+// MARK: - private func
+	private func configureLayout() {
+		self.addSubview(flexRootView)
+
+		flexRootView.flex
+			.direction(.column)
+			.justifyContent(.spaceBetween)
+			.define { flex in
+				flex.addItem(titleLabel)
+				flex.addItem(textField).height(53)
+		 }
+	}
```

Comment:
![image](https://github.com/mash-up-kr/MOIT-iOS/assets/42825223/e212b388-c716-4586-939b-1d2dcab12309)

레이아웃이 약간 깨지는 것 같아요.. ! MOITTextField의 bounds size보다 textfield가 더 밑에 있는 느낌? (튀어나온 느낌 ..?!)
아래처럼 수정하는건 어떨까요? (textfield와 label사이 간격이 10이어야하니 marginTop으로 간격을 주고, 이 둘 사이의 layout은 justifyContent의 default value인 start로 주는거죠 !) 그럼 내부 뷰들의 높이 총 합이 85이 되어 디자이너분이 만들어주신 사이즈가 될 것 같아요 !
```suggestion
	override public func layoutSubviews() {
		super.layoutSubviews()

		flexRootView.pin.all()
		flexRootView.flex.layout()
	}

// MARK: - private func
	private func configureLayout() {
		self.addSubview(flexRootView)

		flexRootView.flex
			.direction(.column)
			.define { flex in
                flex.addItem(titleLabel).height(22)
                
				flex.addItem(textField).height(53)
                    .marginTop(10)
		 }
	}
```
![image](https://github.com/mash-up-kr/MOIT-iOS/assets/42825223/30b502d0-41a1-47e1-a82b-92a976438c0e)
참고부탁드립니당 ~

Reply:
헉 그 부분까지는 확인을 못했네요! 말씀해주신 부분 참고해서 수정해놓도록 하겠습니다🫡